src/main/java/frc/robot/RobotContainer.java

```diff
@@ -41,11 +44,17 @@ public RobotContainer() {
   }
 
   private void configureDefaultCommands() {
+    // Drivebase default command
     Trigger leftBumper = m_driverXboxController.leftBumper();
     Drive driveCommand = new Drive(m_drivebaseSubsystem, m_driverXboxController::getLeftX,
         m_driverXboxController::getLeftY, m_driverXboxController::getRightX, leftBumper::getAsBoolean);
-
     m_drivebaseSubsystem.setDefaultCommand(driveCommand);
+
+    // Claw default command
+    Trigger aButton = m_driverXboxController.a();
+    Trigger bButton = m_driverXboxController.b();
+    ActuateClaw actuationCommand = new ActuateClaw(m_clawSubsystem, aButton::getAsBoolean, bButton::getAsBoolean);
```

Comment:
Instead of passing the buttons into the claw command like we did with the subsystem, focus on having the buttons trigger the behaviors.  In previous years, we had `whileHeld` for this purpose.  For this year, we have `aButton.whileTrue(<command>)` to run a command when a value is true.

Refactor the code to use `whileTrue` and rely on the default command to stop the actuator once the command ends (when the button is no longer pressed).

Reply:
Fixed it